rpc/did/Cargo.toml

```
@@ -1,15 +1,15 @@
 [package]
 edition = "2021"
 name = "did-rpc"
-version = "1.6.2"
+version = "1.7.2"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "3.1.5" }
 jsonrpsee = { version = "0.15.1", features = ["server", "macros"] }
 
-did-rpc-runtime-api = {version = "1.6.2", path = "./runtime-api"}
+did-rpc-runtime-api = {version = "1.7.2", path = "./runtime-api"}
```

we don't need to specify a version here. Maybe we should delete that?
```suggestion
did-rpc-runtime-api = {path = "./runtime-api"}
```

---

I removed the version specification in #413

---

@wischli  alright, I will then not address this comment here. Please make sure to check for any outstanding version specifications once you merge `develop` back.